suite-common/icons/src/icons.ts

```diff
@@ -103,15 +103,14 @@ export const cryptoIcons = {
     name: require('../assets/cryptoIcons/name.svg'),
     nmc: require('../assets/cryptoIcons/nmc.svg'),
     regtest: require('../assets/cryptoIcons/regtest.svg'),
+    sol: require('../assets/cryptoIcons/sol.svg'),
     tada: require('../assets/cryptoIcons/tada.svg'),
     test: require('../assets/cryptoIcons/test.svg'),
-    tsep: require('../assets/cryptoIcons/tsep_tgor.svg'),
-    tgor: require('../assets/cryptoIcons/tsep_tgor.svg'),
+    tsep_tgor: require('../assets/cryptoIcons/tsep_tgor.svg'),
```

Comment:
sorry, there is probably some mistake in `generate-icons` script, but this change is unintentional, I guess. Could you please revert this change? feel free to force push it and I'll merge it

Reply:
nevermind, it has been fixed in https://github.com/trezor/trezor-suite/pull/9239